frontends/election-manager/src/app.test.tsx

```diff
@@ -339,7 +340,8 @@ test('L&A (logic and accuracy) flow', async () => {
   expect(mockKiosk.log).toHaveBeenCalledWith(
     expect.stringContaining(LogEventId.TallyReportPrinted)
   );
-  expect(screen.getAllByText('0').length).toBe(40);
+  const zeroReport = document.querySelector('#zero-report') as HTMLElement;
```

Comment:
Can we give it a `data-testid` and do something like this:

```suggestion
  const zeroReport = screen.getByTestId('zero-report');
```